packages/core/src/payment/payment.ts

```diff
@@ -63,6 +64,22 @@ export interface CreditCardInstrument {
     browser_info?: BrowserInfo;
 }
 
+export interface UsBankAccountInstrument {
```

Comment:
There is no reason to duplicate the interface, because it can be imported from `payment-integration-api` package

Reply:
Thanks. I have removed unnecessary interface